src/plugins/vis_type_timeseries/public/application/components/vis_editor.js

```diff
@@ -76,6 +76,19 @@ export class VisEditor extends Component {
     });
   }, VIS_STATE_DEBOUNCE_DELAY);
 
+  fetchFields = debounce(
```

Comment:
Can we give this function a different name in order to not be confused with the `fetchFields` from lib?

Reply:
Sure, renamed it to `debouncedFetchFields`

Reply:
Much better, thanx!